### DISCUSSION TITLE
Force simple query protocol

### DISCUSSION THREAD

**Discussion by adriangb** [psycopg/psycopg#808] [Q&A]
I want to query pgbouncer to collect stats but the admin interface doesn't support the extended query protocol. Is there any way to force psycopg to use the simple query protocol?

**Comment by dvarrazzo** [comment]
I can't quite remember, but if you use a ClientCursor I think that psycopg
will use the simple query profotocol.

Can you give it a try?

**Reply by adriangb** [reply]
I had to use a combination of `ClientCursor` _and_ `autocommit=True`. Thanks!

```
import psycopg

with psycopg.Connection.connect('...', cursor_factory=psycopg.ClientCursor) as conn:
    conn.autocommit = True
    with conn.cursor() as cur:
        cur.execute('SHOW POOLS')
        print(cur.fetchall())
```

**Reply by dvarrazzo** [reply]
Thank you for the info. I've just checked and probably the need of using autocommit can be relaxed, see #820.
